draft/spec/index.html

```diff
@@ -854,7 +854,17 @@ <h2>Logs Directory</h2>
                 wrong, while others may wish to only store a log entry if an intervention was required.
             </p>
         </blockquote>
+    </section>
 
+    <section id="extensions">
+        <h2>Extensions directory</h2>
+        <p>
+            The base directory of an OCFL Object MAY contain a directory named <code>extensions</code> for the purposes
```

Comment:
I would favor a shorter name, `exts` would be fine for example. I think this not only helps control path lengths but actually makes it easier to talk about the directory in specs vs the extensions that go in them

Reply:
My concern with `exts` is that it is possible to extrapolate different words ("extras"? "externals"?) and it's unlikely that six extra characters will make a difference in path length limits (if you're running that close to the edge you'll likely have other, larger problems). 

The unambiguous nature of the full word `extensions` seems to me to be clearer. We generally apply `code formatting` to talk about directory names to help differentiate between that and extensions in general.

Reply:
"ext" is a common enough root in enough languages that indicates outside-ness that I don't see much confusion. That being said, we have a mechanism for mapping long path names in the inventory anyway so it's not that big an issue.